### PR TITLE
store provider object + scripts

### PR DESCRIPTION
**Pull request by satello**

### CODE REVIEW

**Comment by satello** [comment]
I removed the transpiled js from git. Seems like something unnecessary to track

File: util/StoreProviderWrapper.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+      'POST',
+      this._storeUri + '/' + userAddress,
+      JSON.stringify(Object.assign({}, {contracts: contractsData}, {disputes: disputesData}))
+    )
+    return httpResponse
+  }
+
+  getDocumentsForDispute = async (userAddress, hash) => {
+    const userProfile = await this.getUserProfile(userAddress)
+    if (!userProfile) throw new Error("No profile found for address: " + userAddress)
```
Comment (n1c01a5) [inline util/StoreProviderWrapper.js]: use single quote

File: util/StoreProviderWrapper.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+    let disputeData = _.filter(userProfile.disputes, (o) => {
+      return o.hash === hash
+    })
+
+    if (disputeData.length < 1) return null
```
Comment (n1c01a5) [inline util/StoreProviderWrapper.js]: use lodash `_.isEmpty()`

File: util/StoreProviderWrapper.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+      }
+    })
+  }
+
+  getUserProfile = async (userAddress) => {
+    let httpResponse = await this._makeRequest('GET', this._storeUri + '/' + userAddress)
+    if (httpResponse) {
+      return JSON.parse(httpResponse)
+    } else {
+      return null
```
Comment (n1c01a5) [inline util/StoreProviderWrapper.js]: can be more explicit

File: util/StoreProviderWrapper.js [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+    }
+  }
+
+  newUserProfile = async (userAddress, contractsData = {}, disputesData = {}) => {
+    let httpResponse = await this._makeRequest(
```
Comment (n1c01a5) [inline util/StoreProviderWrapper.js]: use `const`

File: src/kleros.js [hunk 5]
```diff
@@ -41,10 +42,10 @@ class Kleros {
     storeProvider=null
```
Comment (n1c01a5) [inline src/kleros.js]: add spaces

File: script/postDataToStore.js [hunk 6]
```diff
@@ -0,0 +1,24 @@
+import Kleros from '../src/Kleros'
+import Web3 from 'web3'
+import {LOCALHOST_ETH_PROVIDER, LOCALHOST_STORE_PROVIDER} from '../constants'
+import config from '../config'
+
+const contractDataFake = [{"hash": "0x139963ddb98576ddd5f39c14b6cf2c12670b38a1", "contentDocument": '{"evidenceB": "<some byte string>"}'}]
```
Comment (n1c01a5) [inline script/postDataToStore.js]: use single quotes

File: util/StoreProviderWrapper.js [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+        if (httpRequest.readyState === 4) {
+          resolve(JSON.parse(httpRequest.responseText))
+        }
+      }
+      httpRequest.send(body)
+    })
+  }
+
+  getUserProfile = async (userAddress) => {
+    let httpResponse = await this._makeRequest('GET', this._storeUri + '/' + userAddress)
```
[the remark this answers is not in the document]
Comment (n1c01a5) [reply util/StoreProviderWrapper.js]: use `const`

**Pull request merged**